cmd/mirror-main.go

```diff
@@ -70,6 +70,14 @@ var (
 			Name:  "exclude",
 			Usage: "Exclude file/object that matches the passed file name pattern.",
 		},
+		cli.IntFlag{
+			Name:  "older-than",
+			Usage: "Select objects older than N days",
+		},
+		cli.IntFlag{
+			Name:  "newer-than",
+			Usage: "Select objects newer than N days",
```

Comment:
Should we have a common place for these flag definitions and reuse it everywhere?

Reply:
> Should we have a common place for these flag definitions and reuse it everywhere?

In general, I prefer having a specific documentation for each case: mirroring, copying and removing.. for a better documentation clarity. I won't worry too much about duplication here.